cmd/run.go

```diff
@@ -71,15 +71,6 @@ func RunCmd() *cobra.Command {
 			defer stop()
 
 			var mts []agent.Monitor
```

Comment:
we no longer requiring a list now since only otel supporterd?

Reply:
@haveiss this I wanted to discuss with you, just kept it list for extensibility and future support
if we want to add another metrics other than otel in the future, not much code change would be required
any suggestion on this?

Reply:
I don't think we will need another one in foreseeable future, OpenTelemetry is already became a standard, it is quite extensible to anything we need

Reply:
Okay sure, removing the list